constitution.txt

```diff
@@ -598,3 +598,25 @@ The financial year of the association is:
 
 Note: Schedule 1 of the Act provides that an association’s constitution is to address the association’s financial year.
 
+
+43. Objects of the association
+
+(1) The objects for which the association is established are:
+
+  (a) to facilitate the communication of information relating to inventions, results, techniques, and governance of open source;
+
+  (b) to support open source projects and the creators of open source ;
+
+  (c) to support and promote the adoption of open source;
+
+  (d) to support and promote awareness and understanding of open source;
+
+  (e) to conduct open source advocacy with the government, business and civil society;
+
+  (f) to create a network for related local and online open source communities.
+
+(2) The association must pursue charitable purposes only and must apply its income in promoting those purposes.
+
+(3) The association must manage payments and gifts to the association and ensure that they are used to further the stated objects of the association.
+
+(4) In the event of the Organisation being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the Organisation.
```

Comment:
> Should this be association rather than Organisation.

Yep.

```suggestion
(4) In the event of the association being wound up, its assets must go to a charitable entity whose objectives aren’t inconsistent with the stated objects of the association.
```